server/routes/maintainingReports/coordinator.ts

```diff
@@ -1309,11 +1310,36 @@ export default class CoordinatorRoutes {
 
     const data = { reportId, statementId, displayName: staffMember.name, removalRequest }
 
-    res.render('pages/coordinator/reason-for-deleting-this-person.njk', {
-      errors,
-      data,
-      backlinkHref: paths.viewInvolvedStaff(reportId),
-    })
```

Comment:
confirmDeleteStatement now handles both scenarios, ie featureFlagReportEditingEnabled is true and also when false